### PR TITLE
MWPW-134948 use placeholder for toast message

### PR DESCRIPTION
**Pull request by vhargrave**
Fixes the "copied to clipboard" text in the toast to display in proper language

Resolves: [MWPW-134948](https://jira.corp.adobe.com/browse/MWPW-134948)

to test click on the share button on a template
image

Test URLs:
- Before: https://main--express--adobecom.hlx.page/jp/express/?lighthouse=on
- After: https://mwpw-135067-template-toast-text-fix--express--adobecom.hlx.page/jp/express/?lighthouse=on

### CODE REVIEW

**Review by apganapa — APPROVED** [review]

File: express/blocks/template-x/template-rendering.js [hunk 1]
```diff
@@ -351,7 +351,7 @@ function renderStillWrapper(template) {
   const freeTag = createTag('span', { class: 'free-tag' });
 
   if (isFree) {
-    freeTag.append('Free');
+    freeTag.append(placeholders.free ?? 'Free');
```
Comment (JingleH) [inline express/blocks/template-x/template-rendering.js:354]: let's call it "tag-free" for consistency

**Review by JingleH — CHANGES_REQUESTED** [review]

**Review by qiyundai — APPROVED** [review]

**Review by apganapa — APPROVED** [review]

File: express/blocks/template-x/template-rendering.js (hunk 1 above)
Comment (apganapa) [reply express/blocks/template-x/template-rendering.js:354]: Do we have time for this change till tomorrow or we need to get it merged today? @JingleH  @vhargrave

Comment (JingleH) [reply express/blocks/template-x/template-rendering.js:354]: change of heart @vhargrave , let's use generic word "free". Maybe we get to use them elsewhere in the future

**Review by JingleH — APPROVED** [review]

**Review by JingleH — APPROVED** [review]

**Comment by JingleH** [comment]
Just did a merge from main to resolve conflicts

**Review by qiyundai — APPROVED** [review]

**Pull request merged**